client/my-sites/site-settings/controller.js

```diff
@@ -111,38 +99,29 @@ module.exports = {
 	importSite( context ) {
 		renderPage(
 			context,
-			<SiteSettingsComponent sites={ sites } section="import" />
+			<SiteSettingsComponent section="import" />
 		);
 	},
 
 	exportSite( context ) {
 		renderPage(
 			context,
-			<SiteSettingsComponent sites={ sites } section="export" />
+			<SiteSettingsComponent section="export" />
 		);
 	},
 
 	guidedTransfer( context ) {
 		renderPage(
 			context,
-			<SiteSettingsComponent sites={ sites } section="guidedTransfer" hostSlug={ context.params.host_slug } />
+			<SiteSettingsComponent section="guidedTransfer" hostSlug={ context.params.host_slug } />
 		);
 	},
 
 	deleteSite( context ) {
-		let site = sites.getSelectedSite();
```

Comment:
Before getting rid of this in `<DeleteSite`, it has to be refactored, because it will fail when deleting a site. Currently, it uses the sites list actions to delete the site, and if you pass a Redux site object, it'll not remove the site from sites-list, creating odd behavior. So I suggest tackling that first, and then migrating this one.